README.md

```diff
@@ -123,7 +123,9 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
```

Comment:
There should not be any changes to the spacing in this file.

Reply:
Hi Bekah I believe I resolved the requested change to the README.md file. Let me know if there are anymore changes you need from me. Thank you!